static/src/javascripts/projects/mma-tabs/digitalpack-tab.js

```diff
@@ -0,0 +1,107 @@
+define([
+    'bean',
+    'common/utils/$',
+    'common/utils/ajax',
+    'common/utils/config',
+    'mmaTabs/formatters',
+    'mmaTabs/payment-form'
+], function (bean, $, ajax, config, formatters, PaymentForm) {
+
+    var PACKAGE_COST = '.js-dig-package-cost',
+        PAYMENT_FORM = '.js-dig-card-details',
+        SUBSCRIBER_ID = '.js-dig-subscriber-id',
+        REMAINING_TRIAL_LENGTH = '.js-dig-remaining-trial-length',
+        REMAINING_TRIAL_LENGTH_CONTAINER = '.js-dig-remaining-trial-length-container',
+        PACKAGE_CURRENT_RENEWAL_DATE = '.js-dig-current-renewal-date',
+        PACKAGE_CURRENT_PERIOD_END = '.js-dig-current-period-end',
+        PACKAGE_CURRENT_PERIOD_START = '.js-dig-current-period-start',
+        PACKAGE_NEXT_PAYMENT_DATE = '.js-dig-next-payment-date',
+        PACKAGE_NEXT_PAYMENT_PRICE = '.js-dig-next-payment-price',
+        PACKAGE_NEXT_PAYMENT_CONTAINER = '.js-dig-next-payment-container',
+        PACKAGE_INTERVAL = '.js-dig-plan-interval',
+        DETAILS_JOIN_DATE = '.js-dig-join-date',
+        NOTIFICATION_CANCEL = '.js-dig-cancel-tier',
+        DIGITALPACK_DETAILS = '.js-dig-details',
+        DIGITALPACK_PRODUCT = '.js-dig-product',
+        UP_SELL = '.js-dig-up-sell',
+        DIG_INFO = '.js-dig-info',
+        LOADER = '.js-dig-loader',
+        IS_HIDDEN_CLASSNAME = 'is-hidden',
+        stripeForm = new PaymentForm();
+
+    function fetchUserDetails() {
+        ajax({
```

Comment:
There's actually a note in `ajax.js` about it being deprecating and preferring `ajax-promise.js` instead. Might be a good opportunity to switch. https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/utils/ajax.js#L12